auth-manifest/README.md

```diff
@@ -3,37 +3,50 @@
 
 - ### **Image Metadata Entry**
-| Field | Size (bytes) | Description|
-|-------|--------|------------|
-| Image Hash | 48 | SHA2-384 hash of a SOC image |
-| Image Source | 4 | <TBD> |
+
+| Field                   | Size (bytes) | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| ----------------------- | ------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Image Hash              | 48           | SHA2-384 hash of a SOC image                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
+| Image Identifier        | 4            | Unique value selected by the vendor to distinguish between component images.<br />IMEs should be sorted by an ascending order of Image Identifier within an IMC.                                                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
This is not a requirement. Runtime will sort the entries internally for faster lookup. The only requirement is that these Ids should be unique.

Reply:
got it. ill update